SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -203,6 +210,10 @@ bool ValidateIdentifierNames(TArray<TSharedPtr<FUnrealType>>& TypeInfos)
 		{
 			bSuccess = false;
 		}
+		else
+		{
+			CheckWarnAboutRename(SchemaName, ClassName, ClassPath, TEXT("Class"));
```

Comment:
This will not fire, because we prepend ZZ before this point, right?

Reply:
It does fire because it checks `ClassName`, not `SchemaName`. I was on the fence about this. Initially I wanted to add this to the conversion function but that's pretty awkward because we don't have access to the log category and we might not always want to warn in each invocation. So I decided to extract the warning and do it here.

Reply:
I see, at first I was confused, this will fire every time the schema name is valid, and then does the internal check for the class name.

Reply:
Moved the warning into the conversion function now